dev-manual/api/api-reference-storage-service.rst

```diff
@@ -540,13 +544,330 @@ Where payload.json contains::
+
+``api_key``             API key for Dataverse instance.
+
+``agent_name``          A string that will be used in Archivematica METS file to
+                        identify this Dataverse instance as a PREMIS agent,
+                        e.g. My Institution's Dataverse Instance.
+
+``agent_type``          A string that will be used in Archivematica METS file to
+                        identify the type of PREMIS agent named above,
+                        e.g. organization.Name or IP of the remote machine.
+
+``agent_identifier``    A string that will be used as an identifier in the Archivematica
+                        METS file to uniquely identify the PREMIS agent named above.
+
+=====================   ===============================================================
+
+Duracloud
+^^^^^^^^^
+
+Archivematica can use DuraCloud as an access protocol for the Storage Service in
+version 0.5 and higher. Basic fields for a local file system space are listed
+`above \
+<https://www.archivematica.org/en/docs/archivematica-1.15/dev-manual/api/api-reference-storage-service/#create-space>`_
+
+Request body parameters:
+
+==================   =======================================================================
+
+``host``             The hostname of the Duracloud web instance or IP address
+                     with port, e.g. duracloud.example.com:8443
+
+``user``             The username for the Archivematica user that you created in DuraCloud.
+
+``password``         The password for the Archivematica user that you created in DuraCloud.
+
+``duraspace``        The name of the space within DuraCloud.
+
+==================   =======================================================================
+
+DSpace via SWORD2 API
+^^^^^^^^^^^^^^^^^^^^^
+
+DSpace via SWORD2 locations are currently supported only for AIP Storage locations.
+Basic fields for a local file system space are listed `above \
+<https://www.archivematica.org/en/docs/archivematica-1.15/dev-manual/api/api-reference-storage-service/#create-space>`_
+
+Request body parameters:
+
+===================     =======================================================================================
+
+``sd_iri``              URL of the service document. E.g. ``http://demo.dspace.org/swordv2/servicedocument,``
+                        where servicedocument is the handle to the community or collection being used
+                        for deposit.
+
+``user``                A username for the DSpace instance with sufficient permissions to permit
+                        authentication.
+
+``password``            The password for the Dspace user that you created.
+
+``metadata_policy``     Use to restrict access to the metadata bitstream. Must be specified as a list of
+                        objects in JSON, e.g. ``[{"action":"READ","groupId":"5","rpType":"TYPE_CUSTOM"}]``.
+
+``archive_format``      Select Archive format from dropdown. e.g. ``ZIP or 7z``
```

Comment:
What dropdown does this refer to? Would it be enough to just list the values of the field choices?

```suggestion
``archive_format``      Archive format, e.g. ``ZIP``, ``7z``.
```